taipy/common/config/__init__.py

```diff
@@ -9,9 +9,7 @@
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
-"""# Taipy `config` Package
-
-The `taipy.common.config` package provides features to configure a Taipy application.
+""" The `taipy.common.config` package provides features to configure a Taipy application.
```

Comment:
No title like '# Taipy Configuration package'?

Reply:
Nop. If no title is defined here, the default title added by the refman step is : 

"taipy.common.config package"

Which is enough for me.